--- progsreiserfs-0.3.1-rc8/progsreiserfs/cpfs/cpfs.c.orig	2002-12-12 13:32:32.000000000 +0100
+++ progsreiserfs-0.3.1-rc8/progsreiserfs/cpfs/cpfs.c	2003-12-28 04:29:04.207074616 +0100
@@ -58,6 +58,12 @@
 	{0, 0, 0, 0}
     };
 	
+#ifdef ENABLE_NLS
+    setlocale(LC_ALL, "");
+    bindtextdomain(PACKAGE, LOCALEDIR);
+    textdomain(PACKAGE);
+#endif
+	
     while ((choice = getopt_long_only(argc, argv, "uvj:nq", long_options, 
 	(int *)0)) != EOF) 
     {
